haproxy/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ this is the changelog for Server Templates
 
 v2.0.2
 -----
+- fixing sticky session false to nil 
+
+v2.0.2
```

Comment:
This change should be above the v2.0.2 version and this version should be v2.0.3

Reply:
Updated